### PR TITLE
feat(network): add mantle network

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1551,6 +1551,21 @@
...
+    "network": "mainnet",
+    "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "rpc": [
+      "https://rpc.mantle.xyz"
+    ],
+    "explorer": {
+      "url": "https://explorer.testnet.mantle.xyz"
+    },
+    "start": 168863,
+    "logo": ""
```
Comment (bonustrack) [inline src/networks.json:1567]: We need to logo too, you can take the image from Twitter

File: src/networks.json [hunk 2]
```diff
@@ -1551,6 +1551,21 @@
...
     "logo": "ipfs://QmNkr1UPcBYbvLp7d7Pk4eF3YCsHpaNkfusKZNtykL2EQC"
   },
+  "5000": {
+    "key": "5000",
+    "name": "Mantle",
+    "chainId": 5000,
+    "network": "mainnet",
+    "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "rpc": [
+      "https://rpc.mantle.xyz"
```
Comment (bonustrack) [inline src/networks.json:1561]: Did you tried if this is a full archive node?

File: src/networks.json [hunk 3]
```diff
@@ -1551,6 +1551,21 @@
...
+    ],
+    "explorer": {
+      "url": "https://explorer.testnet.mantle.xyz"
```
Comment (bonustrack) [inline src/networks.json:1564]: This is not the testnet
```suggestion
      "url": "https://explorer.mantle.xyz"
```

File: src/networks.json (hunk 2 above)
Comment (Todmy) [reply src/networks.json:1561]: yes
Screenshot 2023-08-23 at 12 15 07

**Review by bonustrack — DISMISSED** [review]
utACK

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**